From ba9b81d9dd0788870d1b5ba9e1ec029fffde09d0 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@notvalid.com>
Date: Mon, 19 Oct 2026 22:07:14 -0700
Subject: [PATCH] fixup! v2: swap CTransaction and BlockMerkleRoot to V2 v7
 wire format

Restore all27 inherited legacy CLI vectors byte-for-byte from the exact pre-native owner parent. Legacy transaction identity is SHA256d again, and bitcoin-tx continues to create ordinary version2 transactions. Independently checked paired JSON txid/hash/raw-size values against SHA256d of their hex. Complete util CLI test passes; source/reference hashes in final-sweep/cli-legacy-vector-restoration.json. Tier: V2
---
 test/util/data/bitcoin-util-test.json     | 4 ++--
 test/util/data/blanktxv1.json             | 4 ++--
 test/util/data/blanktxv2.hex              | 2 +-
 test/util/data/blanktxv2.json             | 8 ++++----
 test/util/data/tt-delin1-out.json         | 4 ++--
 test/util/data/tt-delout1-out.json        | 4 ++--
 test/util/data/tt-locktime317000-out.json | 4 ++--
 test/util/data/txcreate1.hex              | 2 +-
 test/util/data/txcreate1.json             | 8 ++++----
 test/util/data/txcreate2.hex              | 2 +-
 test/util/data/txcreate2.json             | 8 ++++----
 test/util/data/txcreatedata1.hex          | 2 +-
 test/util/data/txcreatedata1.json         | 4 ++--
 test/util/data/txcreatedata2.hex          | 2 +-
 test/util/data/txcreatedata2.json         | 8 ++++----
 test/util/data/txcreatedata_seq0.hex      | 2 +-
 test/util/data/txcreatedata_seq0.json     | 8 ++++----
 test/util/data/txcreatedata_seq1.json     | 4 ++--
 test/util/data/txcreatemultisig1.json     | 4 ++--
 test/util/data/txcreatemultisig2.json     | 4 ++--
 test/util/data/txcreateoutpubkey1.json    | 4 ++--
 test/util/data/txcreatescript1.json       | 4 ++--
 test/util/data/txcreatescript2.json       | 4 ++--
 test/util/data/txcreatescript5.hex        | 2 +-
 test/util/data/txcreatescript6.hex        | 2 +-
 test/util/data/txcreatesignv1.json        | 4 ++--
 test/util/data/txcreatesignv2.hex         | 2 +-
 27 files changed, 55 insertions(+), 55 deletions(-)

diff --git a/test/util/data/bitcoin-util-test.json b/test/util/data/bitcoin-util-test.json
index 6428b4411f..d3789ca288 100644
--- a/test/util/data/bitcoin-util-test.json
+++ b/test/util/data/bitcoin-util-test.json
@@ -200,12 +200,12 @@
     "description": "Creates a new transaction with a single empty output script (output in json)"
   },
   { "exec": "./bitcoin-tx",
-    "args": ["07000080000100000000000000000000000000"],
+    "args": ["02000000000100000000000000000000000000"],
     "output_cmp": "txcreate2.hex",
     "description": "Parses a transaction with no inputs and a single output script"
   },
   { "exec": "./bitcoin-tx",
-    "args": ["-json", "07000080000100000000000000000000000000"],
+    "args": ["-json", "02000000000100000000000000000000000000"],
     "output_cmp": "txcreate2.json",
     "description": "Parses a transaction with no inputs and a single output script (output in json)"
   },
diff --git a/test/util/data/blanktxv1.json b/test/util/data/blanktxv1.json
index ae38ad3357..c434cf4a9e 100644
--- a/test/util/data/blanktxv1.json
+++ b/test/util/data/blanktxv1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "374fa10488ee7321f55fcd9d41a2db95b5d93857b49a83c319acc9c2e1d79d46",
-    "hash": "fc2bb4238506ad81117386d0b06eb2a844e6eb0db78f36d246f1dc117ddacac0",
+    "txid": "d21633ba23f70118185227be58a63527675641ad37967e2aa461559f577aec43",
+    "hash": "d21633ba23f70118185227be58a63527675641ad37967e2aa461559f577aec43",
     "version": 1,
     "size": 10,
     "locktime": 0,
diff --git a/test/util/data/blanktxv2.hex b/test/util/data/blanktxv2.hex
index db7e144ef4..22d830eda1 100644
--- a/test/util/data/blanktxv2.hex
+++ b/test/util/data/blanktxv2.hex
@@ -1 +1 @@
-07000080000000000000
+02000000000000000000
diff --git a/test/util/data/blanktxv2.json b/test/util/data/blanktxv2.json
index 175b8afb3a..43400425b5 100644
--- a/test/util/data/blanktxv2.json
+++ b/test/util/data/blanktxv2.json
@@ -1,12 +1,12 @@
 {
-    "txid": "8a92c58de90e74ad8d69cc8c0007b87b72bdb33e8d8448aa048932a2421cfc4b",
-    "hash": "7ca00798c2f1cbcb47490abea5ee3196f3fbc82b85aa910fdbf9de41179947ab",
-    "version": 7,
+    "txid": "4ebd325a4b394cff8c57e8317ccf5a8d0e2bdf1b8526f8aad6c8e43d8240621a",
+    "hash": "4ebd325a4b394cff8c57e8317ccf5a8d0e2bdf1b8526f8aad6c8e43d8240621a",
+    "version": 2,
     "size": 10,
     "locktime": 0,
     "vin": [
     ],
     "vout": [
     ],
-    "hex": "07000080000000000000"
+    "hex": "02000000000000000000"
 }
diff --git a/test/util/data/tt-delin1-out.json b/test/util/data/tt-delin1-out.json
index 654624e11f..6b54272349 100644
--- a/test/util/data/tt-delin1-out.json
+++ b/test/util/data/tt-delin1-out.json
@@ -1,6 +1,6 @@
 {
-    "txid": "2588b5f5f27786494ec7fa92d49374920e6a0ac2c4a95503123f9859e3f700af",
-    "hash": "544e569799252e4af26f492df685ce4f6a97543a9307bed8b0f8681253bf30c6",
+    "txid": "81b2035be1da1abe745c6141174a73d151009ec17b3d5ebffa2e177408c50dfd",
+    "hash": "81b2035be1da1abe745c6141174a73d151009ec17b3d5ebffa2e177408c50dfd",
     "version": 1,
     "size": 3040,
     "locktime": 0,
diff --git a/test/util/data/tt-delout1-out.json b/test/util/data/tt-delout1-out.json
index 315cd28bf4..43b5052a75 100644
--- a/test/util/data/tt-delout1-out.json
+++ b/test/util/data/tt-delout1-out.json
@@ -1,6 +1,6 @@
 {
-    "txid": "8f9c8cb74e034c35e48144cf35dd11fa39b91796b9ffe59e5e34fb8f4f1f4136",
-    "hash": "2c977b66074a1aa6d67fd873e87e29f272cb87f3e1ede15028cd9e1225bf0a61",
+    "txid": "c46ccd75b5050e942b2e86a3648f843f525fe6fc000bf0534ba5973063354493",
+    "hash": "c46ccd75b5050e942b2e86a3648f843f525fe6fc000bf0534ba5973063354493",
     "version": 1,
     "size": 3155,
     "locktime": 0,
diff --git a/test/util/data/tt-locktime317000-out.json b/test/util/data/tt-locktime317000-out.json
index 15078f4129..2a340c4e78 100644
--- a/test/util/data/tt-locktime317000-out.json
+++ b/test/util/data/tt-locktime317000-out.json
@@ -1,6 +1,6 @@
 {
-    "txid": "008d30d9100f41ef5f3cc30f379d724122e5f9639415b8e28df6f47bb19af4cc",
-    "hash": "a36eaf5a60be0ee217711004a85187a915487064c39af8fb629a65849712d31f",
+    "txid": "aded538f642c17e15f4d3306b8be7e1a4d1ae0c4616d641ab51ea09ba65e5cb5",
+    "hash": "aded538f642c17e15f4d3306b8be7e1a4d1ae0c4616d641ab51ea09ba65e5cb5",
     "version": 1,
     "size": 3189,
     "locktime": 317000,
diff --git a/test/util/data/txcreate1.hex b/test/util/data/txcreate1.hex
index 32df485465..9ec6ee3531 100644
--- a/test/util/data/txcreate1.hex
+++ b/test/util/data/txcreate1.hex
@@ -1 +1 @@
-07000080031f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff7cca453133921c50d5025878f7f738d1df891fd359763331935784cf6b9c82bf1200000000fffffffffccd319e04a996c96cfc0bf4c07539aa90bd0b1a700ef72fae535d6504f9a6220100000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0084d717000000001976a914f2d4db28cad6502226ee484ae24505c2885cb12d88ac00000000
+02000000031f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff7cca453133921c50d5025878f7f738d1df891fd359763331935784cf6b9c82bf1200000000fffffffffccd319e04a996c96cfc0bf4c07539aa90bd0b1a700ef72fae535d6504f9a6220100000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0084d717000000001976a914f2d4db28cad6502226ee484ae24505c2885cb12d88ac00000000
diff --git a/test/util/data/txcreate1.json b/test/util/data/txcreate1.json
index 41bc210d1f..d60743d3b0 100644
--- a/test/util/data/txcreate1.json
+++ b/test/util/data/txcreate1.json
@@ -1,7 +1,7 @@
 {
-    "txid": "68c229fb60cd0cf7cf3abf30ebcc453e62327643894ed5080ef5966b2ba6576c",
-    "hash": "ea6e882249c7233fdc5d5c1814eabf0215cd00380acac86939d9388ae8b29f55",
-    "version": 7,
+    "txid": "fe7d174f42dce0cffa7a527e9bc8368956057619ec817648f6138b98f2533e8f",
+    "hash": "fe7d174f42dce0cffa7a527e9bc8368956057619ec817648f6138b98f2533e8f",
+    "version": 2,
     "size": 201,
     "locktime": 0,
     "vin": [
@@ -61,5 +61,5 @@
             }
         }
     ],
-    "hex": "07000080031f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff7cca453133921c50d5025878f7f738d1df891fd359763331935784cf6b9c82bf1200000000fffffffffccd319e04a996c96cfc0bf4c07539aa90bd0b1a700ef72fae535d6504f9a6220100000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0084d717000000001976a914f2d4db28cad6502226ee484ae24505c2885cb12d88ac00000000"
+    "hex": "02000000031f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff7cca453133921c50d5025878f7f738d1df891fd359763331935784cf6b9c82bf1200000000fffffffffccd319e04a996c96cfc0bf4c07539aa90bd0b1a700ef72fae535d6504f9a6220100000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0084d717000000001976a914f2d4db28cad6502226ee484ae24505c2885cb12d88ac00000000"
 }
diff --git a/test/util/data/txcreate2.hex b/test/util/data/txcreate2.hex
index f218855aaf..38bb7b1046 100644
--- a/test/util/data/txcreate2.hex
+++ b/test/util/data/txcreate2.hex
@@ -1 +1 @@
-07000080000100000000000000000000000000
+02000000000100000000000000000000000000
diff --git a/test/util/data/txcreate2.json b/test/util/data/txcreate2.json
index efdc2a4780..6370677fdf 100644
--- a/test/util/data/txcreate2.json
+++ b/test/util/data/txcreate2.json
@@ -1,7 +1,7 @@
 {
-    "txid": "20efb28dd083470c5299d65884152e08657da8d382d786e2c4abd314c93280ec",
-    "hash": "48b32929bfbe3260b98292928be78f439e83348ab0e23b46f066c9c47fd7729a",
-    "version": 7,
+    "txid": "0481afb29931341d0d7861d8a2f6f26456fa042abf54a23e96440ed7946e0715",
+    "hash": "0481afb29931341d0d7861d8a2f6f26456fa042abf54a23e96440ed7946e0715",
+    "version": 2,
     "size": 19,
     "locktime": 0,
     "vin": [
@@ -17,5 +17,5 @@
             }
         }
     ],
-    "hex": "07000080000100000000000000000000000000"
+    "hex": "02000000000100000000000000000000000000"
 }
diff --git a/test/util/data/txcreatedata1.hex b/test/util/data/txcreatedata1.hex
index ba2a4d5b70..cefd1a05a6 100644
--- a/test/util/data/txcreatedata1.hex
+++ b/test/util/data/txcreatedata1.hex
@@ -1 +1 @@
-07000080011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0084d71700000000526a4c4f54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e00000000
+02000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0084d71700000000526a4c4f54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e00000000
diff --git a/test/util/data/txcreatedata1.json b/test/util/data/txcreatedata1.json
index 2db560052f..435da7b805 100644
--- a/test/util/data/txcreatedata1.json
+++ b/test/util/data/txcreatedata1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "a2f3154b495c5815bdd5d0da480bb8db51fafd751bc4bc1ec2f65f4621cb6d60",
-    "hash": "0684b997d7effa4b23a3b3379bf831c164f86899d879f243ffbe39fe031ab18e",
+    "txid": "07894b4d12fe7853dd911402db1620920d261b9627c447f931417d330c25f06e",
+    "hash": "07894b4d12fe7853dd911402db1620920d261b9627c447f931417d330c25f06e",
     "version": 1,
     "size": 176,
     "locktime": 0,
diff --git a/test/util/data/txcreatedata2.hex b/test/util/data/txcreatedata2.hex
index b42e4114a3..d69cf58ba1 100644
--- a/test/util/data/txcreatedata2.hex
+++ b/test/util/data/txcreatedata2.hex
@@ -1 +1 @@
-07000080011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0000000000000000526a4c4f54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e00000000
+02000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0000000000000000526a4c4f54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e00000000
diff --git a/test/util/data/txcreatedata2.json b/test/util/data/txcreatedata2.json
index de9d8fb896..03099ef00c 100644
--- a/test/util/data/txcreatedata2.json
+++ b/test/util/data/txcreatedata2.json
@@ -1,7 +1,7 @@
 {
-    "txid": "ccad1c5e873188eb12cc50749928fc353303713278fab1c13511d1290475486d",
-    "hash": "2daf69153cc0876f7a12ac4db0896868910b5aaa6c8fda62c816ef9b91b17009",
-    "version": 7,
+    "txid": "c14b007fa3a6c1e7765919c1d14c1cfc2b8642c3a5d3be4b1fa8c4ccfec98bb0",
+    "hash": "c14b007fa3a6c1e7765919c1d14c1cfc2b8642c3a5d3be4b1fa8c4ccfec98bb0",
+    "version": 2,
     "size": 176,
     "locktime": 0,
     "vin": [
@@ -39,5 +39,5 @@
             }
         }
     ],
-    "hex": "07000080011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0000000000000000526a4c4f54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e00000000"
+    "hex": "02000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000ffffffff0280a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac0000000000000000526a4c4f54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e00000000"
 }
diff --git a/test/util/data/txcreatedata_seq0.hex b/test/util/data/txcreatedata_seq0.hex
index d1a7542b73..54b89d2381 100644
--- a/test/util/data/txcreatedata_seq0.hex
+++ b/test/util/data/txcreatedata_seq0.hex
@@ -1 +1 @@
-07000080011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000fdffffff0180a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac00000000
+02000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000fdffffff0180a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac00000000
diff --git a/test/util/data/txcreatedata_seq0.json b/test/util/data/txcreatedata_seq0.json
index 7d88f23d95..951683411f 100644
--- a/test/util/data/txcreatedata_seq0.json
+++ b/test/util/data/txcreatedata_seq0.json
@@ -1,7 +1,7 @@
 {
-    "txid": "70725955a5eea590c5d3e855054dfc48160caaff933987ed2333e8fc77ef18e1",
-    "hash": "17103113d6766a590787957193f3861b0f13ea56de766c7522c7588db0b727b7",
-    "version": 7,
+    "txid": "8df6ed527472542dd5e137c242a7c5a9f337ac34f7b257ae4af886aeaebb51b0",
+    "hash": "8df6ed527472542dd5e137c242a7c5a9f337ac34f7b257ae4af886aeaebb51b0",
+    "version": 2,
     "size": 85,
     "locktime": 0,
     "vin": [
@@ -30,5 +30,5 @@
             }
         }
     ],
-    "hex": "07000080011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000fdffffff0180a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac00000000"
+    "hex": "02000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000fdffffff0180a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac00000000"
 }
diff --git a/test/util/data/txcreatedata_seq1.json b/test/util/data/txcreatedata_seq1.json
index f8ba8e157b..ae6de45fa5 100644
--- a/test/util/data/txcreatedata_seq1.json
+++ b/test/util/data/txcreatedata_seq1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "7e28e4c0fd70afb3b16e4a1c85752c6b3df296d3e74963f3fd008d3d64899ea5",
-    "hash": "fc19efb69873c7bbb0c5170bb3025eaf61f6d8e767786744bbcfae1b6d1ecc26",
+    "txid": "c4dea671b0d7b48f8ab10bc46650e8329d3c5766931f548f513847a19f5ba75b",
+    "hash": "c4dea671b0d7b48f8ab10bc46650e8329d3c5766931f548f513847a19f5ba75b",
     "version": 1,
     "size": 126,
     "locktime": 0,
diff --git a/test/util/data/txcreatemultisig1.json b/test/util/data/txcreatemultisig1.json
index 7320155fa4..14d00f5dc6 100644
--- a/test/util/data/txcreatemultisig1.json
+++ b/test/util/data/txcreatemultisig1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "c21ff718c74d26c2faffc6dd682cb122091963bf33c238946fd6a983a3bbe986",
-    "hash": "6de9130121f092fe8384441a609977efe16d01cd60aa9bb06a21fb9263913516",
+    "txid": "0d1d4edfc217d9db3ab6a9298f26a52eae3c52f55a6cb8ccbc14f7c727572894",
+    "hash": "0d1d4edfc217d9db3ab6a9298f26a52eae3c52f55a6cb8ccbc14f7c727572894",
     "version": 1,
     "size": 124,
     "locktime": 0,
diff --git a/test/util/data/txcreatemultisig2.json b/test/util/data/txcreatemultisig2.json
index 1229a5081a..4dda1e63c7 100644
--- a/test/util/data/txcreatemultisig2.json
+++ b/test/util/data/txcreatemultisig2.json
@@ -1,6 +1,6 @@
 {
-    "txid": "effc50a6a1d34f4ff05b8483de18bd2852b4cb8a75e285b145aee9cfdf30ff23",
-    "hash": "0bf8442b0eb1117d6403cce079c0413eeafa3764e4fd89a23106021d9940ca8d",
+    "txid": "0d861f278a3b7bce7cb5a88d71e6e6a903336f95ad5a2c29b295b63835b6eee3",
+    "hash": "0d861f278a3b7bce7cb5a88d71e6e6a903336f95ad5a2c29b295b63835b6eee3",
     "version": 1,
     "size": 42,
     "locktime": 0,
diff --git a/test/util/data/txcreateoutpubkey1.json b/test/util/data/txcreateoutpubkey1.json
index 71c3769e69..55e9de631a 100644
--- a/test/util/data/txcreateoutpubkey1.json
+++ b/test/util/data/txcreateoutpubkey1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "ea605ccb293e5b4dce9e3944b77cdeab6d708c3acf26bd7adc1c9f104132dc14",
-    "hash": "6a6543082e0877beda7875cce8df674d7f823aa50ab574ef9c8f3abe1a950bd1",
+    "txid": "f42b38ac12e3fafc96ba1a9ba70cbfe326744aef75df5fb9db5d6e2855ca415f",
+    "hash": "f42b38ac12e3fafc96ba1a9ba70cbfe326744aef75df5fb9db5d6e2855ca415f",
     "version": 1,
     "size": 54,
     "locktime": 0,
diff --git a/test/util/data/txcreatescript1.json b/test/util/data/txcreatescript1.json
index 5666e19e99..857df3ec9e 100644
--- a/test/util/data/txcreatescript1.json
+++ b/test/util/data/txcreatescript1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "1688c44e8e8326939c8687efab623e2a3d8ccdcec32bed5863f26062797a550d",
-    "hash": "8541643946ff61bc46e2ee9c572831c9502e62e0a1d3c233c8622a5cad39c234",
+    "txid": "f0851b68202f736b792649cfc960259c2374badcb644ab20cac726b5f72f61c9",
+    "hash": "f0851b68202f736b792649cfc960259c2374badcb644ab20cac726b5f72f61c9",
     "version": 1,
     "size": 20,
     "locktime": 0,
diff --git a/test/util/data/txcreatescript2.json b/test/util/data/txcreatescript2.json
index 5044f10530..5ef09d53e5 100644
--- a/test/util/data/txcreatescript2.json
+++ b/test/util/data/txcreatescript2.json
@@ -1,6 +1,6 @@
 {
-    "txid": "8ee5f1f673353134a0c9af0fcb0caffad148e917d2f4343c65567ece389ffc34",
-    "hash": "84e0439f1e4469d6cb1209cd94d93c70b3aa9d286e55735b375da450cad43f1e",
+    "txid": "6e07a7cc075e0703f32ee8c4e5373fe654bfbc315148fda364e1be286ff290d0",
+    "hash": "6e07a7cc075e0703f32ee8c4e5373fe654bfbc315148fda364e1be286ff290d0",
     "version": 1,
     "size": 42,
     "locktime": 0,
diff --git a/test/util/data/txcreatescript5.hex b/test/util/data/txcreatescript5.hex
index 190ee21d9b..e1d5fe6aee 100644
--- a/test/util/data/txcreatescript5.hex
+++ b/test/util/data/txcreatescript5.hex
@@ -1 +1 @@
-07000080000100000000000000000908ffffffffffffff7f00000000
+02000000000100000000000000000908ffffffffffffff7f00000000
diff --git a/test/util/data/txcreatescript6.hex b/test/util/data/txcreatescript6.hex
index 383c27b232..93d3d789e8 100644
--- a/test/util/data/txcreatescript6.hex
+++ b/test/util/data/txcreatescript6.hex
@@ -1 +1 @@
-07000080000100000000000000000908ffffffffffffffff00000000
+02000000000100000000000000000908ffffffffffffffff00000000
diff --git a/test/util/data/txcreatesignv1.json b/test/util/data/txcreatesignv1.json
index edac8551eb..4565742a6e 100644
--- a/test/util/data/txcreatesignv1.json
+++ b/test/util/data/txcreatesignv1.json
@@ -1,6 +1,6 @@
 {
-    "txid": "5beb162fd36eb04834540154f162e1cf9953f0748c3f8cec4650e70003eceaf6",
-    "hash": "7f3ebc9a91ebe11231ceb41d0ca077cd082221226e0441032ecc21a65236d294",
+    "txid": "bea13b5f794dc351d1d890b525b0af423c49ab49c667c8fd95cbba2fb283cd5e",
+    "hash": "bea13b5f794dc351d1d890b525b0af423c49ab49c667c8fd95cbba2fb283cd5e",
     "version": 1,
     "size": 223,
     "locktime": 0,
diff --git a/test/util/data/txcreatesignv2.hex b/test/util/data/txcreatesignv2.hex
index e3de472764..a102a67742 100644
--- a/test/util/data/txcreatesignv2.hex
+++ b/test/util/data/txcreatesignv2.hex
@@ -1 +1 @@
-07000080018594c5bdcaec8f06b78b596f31cd292a294fd031e24eec716f43dac91ea7494d000000008a473044022073b65c0e66181c84940bd939942eb5adbe0bf90b8e374000e5f8ab1170ac89ae02205401fde2b2275eef50794aeb37ac673d2338a8dd71a7080fb5697707d12b710701410479be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8ffffffff01a0860100000000001976a9145834479edbbe0539b31ffd3a8f8ebadc2165ed0188ac00000000
+02000000018594c5bdcaec8f06b78b596f31cd292a294fd031e24eec716f43dac91ea7494d000000008a47304402205338be788a5c8ec16624f60c244927054e7b07bc8e540b14b1f4a16842ddefd102204e4db3e867caa416a6ec592addc30ba379c1e899913caf97214ed3e0aa3c866a01410479be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8ffffffff01a0860100000000001976a9145834479edbbe0539b31ffd3a8f8ebadc2165ed0188ac00000000
-- 
2.50.1 (Apple Git-155)

